easybuild/easyblocks/o/openbabel.py

```diff
@@ -76,7 +92,15 @@ def sanity_check_step(self):
     def make_module_extra(self):
         """Custom variables for OpenBabel module."""
         txt = super(EB_OpenBabel, self).make_module_extra()
-        txt += self.module_generator.prepend_paths('PYTHONPATH', ['lib'])
+        if self.with_python:
+            if LooseVersion(self.version) >= LooseVersion('2.4'):
+                # since OpenBabel 2.4.0 the Python bindings under 
+                # ${PREFIX}/lib/python2.7/site-packages  rather than ${PREFIX}/lib
+                shortpyver = '.'.join(get_software_version('Python').split('.')[:2])
+                ob_pythonpath = 'lib/python%s/site-packages/' % (shortpyver)
```

Comment:
we have a dedicated function for this in the `PythonPackage` generic easyblock, so this can simply become:

```python
ob_pythonpath = det_pylibdir()
```

In some cases it may be `lib64` rather than `lib`, etc., which is taken into account by `det_pylibdir()`